webapp/playbooks/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "playbooks",
   "private": true,
+  "version": "1.36.0",
```

Comment:
Do we need to have the same version though out the products going forward?

Reply:
I think it should just match the web app version for now. I'll try to remember to keep these up to date when I bump the versions of the types and client packages since those matter more because they're shipped to the world

Reply:
I bumped up the same version of webapp and Playbooks. I have bumped up boards to the same as well.